svmbir/phantom.py

```diff
@@ -62,10 +63,10 @@ def gen_microscopy_sample(num_rows, num_cols):
         {'x0': 0.25, 'y0': -0.8, 'a': 0.2, 'b': 0.08, 'theta': 0, 'gray_level': 0.04},
         {'x0': -0.04, 'y0': -1.3, 'a': 0.33, 'b': 0.15, 'theta': 0, 'gray_level': 0.04}
     ]
+    axis_cols = np.linspace(-1, 1, num_rows)
+    axis_rows = np.linspace(2, -2, num_cols)
 
-    axis_rows = np.linspace(-1, 1, num_cols)
-    axis_cols = np.linspace(-2, 2, num_rows)
-    x_grid, y_grid = np.meshgrid(axis_rows, -axis_cols)
+    x_grid, y_grid = np.meshgrid(axis_cols, axis_rows )
```

Comment:
In gen_microscopy_sample, when I run below,

`  

    image = gen_microscopy_sample(256,128)

    plt.imshow(image,vmin=1,vmax=1.1)

    plt.savefig('ms.png')
`

I will get the below image. 
![ms](https://user-images.githubusercontent.com/22163570/99346213-20f89480-2862-11eb-85bf-852a51538e60.png)

In 'x_grid, y_grid = np.meshgrid(axis_cols, axis_rows ) ', axis_cols and axis_rows should switch.